db/migrate/20170703173452_add_miq_expression_enable_and_text_disable.rb

```diff
@@ -0,0 +1,6 @@
+class AddMiqExpressionEnableAndTextDisable < ActiveRecord::Migration[5.0]
+  def change
+    add_column :custom_buttons, :miq_expression_enable, :text
+    add_column :custom_buttons, :text_disable, :text
```

Comment:
I'm thinking `disabled_text` sounds a little better.  Otherwise it reads to me like "whether or not to disable the text portion of the button".